@xen-orchestra/web/src/components/storage-repositories/panel/table/StorageRepositorySizeProgressCell.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="progress-cell">
+    <VtsProgressBar :current :total label="" noruler class="progress" />
```

Comment:
If the label is not necessary, maybe make it optional and remove the label here ? 
```suggestion
    <VtsProgressBar :current :total noruler class="progress" />
```